Specs/DateTime/English/DateTimeModel.json

```diff
@@ -14025,6 +14025,68 @@
         }
       }
     ]
+  },
+  {
+    "Input": "6,107.31 August 2019 should not include the decimal",
+    "Comment": "Something wrong with negative lookbehind in java",
+    "NotSupported": "java",
+    "Results": [
+      {
+        "Text": "august 2019",
+        "Start": 9,
+        "End": 19,
+        "TypeName": "datetimeV2.daterange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2019-08",
+              "type": "daterange",
+              "start": "2019-08-01",
+              "end": "2019-09-01"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "feb 30 2019",
+    "Results": [
+      {
+        "Text": "feb 30 2019",
+        "Start": 0,
+        "End": 10,
+        "TypeName": "datetimeV2.date",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2019-02-30",
+              "type": "date",
+              "value": "not resolved"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "feb 30",
```

Comment:
Please also add the full datetime case: "That's absurd like saying we should meet in feb 30 2019 17:20!"